packages/Button/src/Button.styles.js

```diff
@@ -266,6 +266,13 @@ function getIconColor(props) {
   return iconColors[props.kind];
 }
 
+export const GlobalStyle = createGlobalStyle`
+  [data-whatinput="mouse"] [data-pka-type="button"]:focus {
```

Comment:
You don't need global styles here. And `[data-pka-type="button"]` too.

You can add this styles to `buttonStyles`:
```css
[data-whatinput="mouse"] &:focus {
  ...
}
```